### PR TITLE
Some improvements : keep the original /etc/mongod.conf, update the /etc/init.d/mongod and check dbpath dir rights

### PR DESCRIPTION
**Pull request by jmcollin78**
cf. title

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# MongoDB Cookbook
+# MongoDB Cookbook for by JMC
+Note: this fork has been created to create pull request in chef-brigade Repo. Do not use directly when PR are integrated.
```
Comment (Niluge-KiWi) [inline README.md]: cannot merge this

Comment (jmcollin78) [inline README.md]: Yes of course. Can you merge all but rhe README ?
Le 6 oct. 2015 3:03 PM, "Thomas Riccardi" <notifications@github.com> a
écrit :

**Comment by jmcollin78** [comment]
I've remove the extra line in README.

File: recipes/replicaset.rb [hunk 2]
```diff
@@ -25,7 +25,9 @@
 ruby_block 'chef_gem_at_converge_time' do
   block do
     node['mongodb']['ruby_gems'].each do |gem, version|
+      Chef::Log.info("jmcnet/mongodb-coookbook : installing gem #{gem} with version #{version}")
```
Comment (luishdez) [inline recipes/replicaset.rb]: `jmcnet/mongodb-coookbook` ??

**Comment by luishdez** [comment]
I've to check this, but at first sight I think it will break things for other setups… and Travis is failing for 1.9.3

**Comment by luishdez** [comment]
I've to check this, but at first sight it will break things and Travis is failing for 1.9.3

**Comment by jmcollin78** [comment]
I try to understand what is problem with Travis and I can't figure out why it fails. May be you can help me. Here is the error:
<code>Gem::InstallError: berkshelf-api-client requires Ruby version >= 2.0.0.
An error occurred while installing berkshelf-api-client (1.4.0), and Bundler
</code>

**Comment by shortdudey123** [comment]
Closing since most of this is addressed in the current v1 rewrite
Please open a new PR based off of current master if you feel this is still relevant

**Pull request closed without merging**